readme.md

```diff
@@ -176,4 +176,40 @@ Respond to GET requests and show course information according to specific course
   "location": "GLFN AUD",
   "type": "Lecture"
 }
+```
+
+## DELETE
+
+Delete a course object from the database.
+
+#### DELETE by CRN
+
+Respond to DELETE requests and delete course object according to specific course CRN, or return not found message if the course crn is invalid.
```

Comment:
Your implementation looks good, but it doesn't match your documentation. Your delete api endpoint will always return ok, but that's not what your documentation says :)